### PR TITLE
fix: new modules to match the code style + add logging module mock for…

### PR DESCRIPTION
**Pull request by shine2lay**
### Context
Some of the new classes introduce wasn't matching the code style of the repo, this PR is meant to fix that plus a few other minor improvements.

### CODE REVIEW

File: __fixtures__/setup/jestSetUp.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    create: () => {
+      return {
+        info: jest.fn(),
+        warn: jest.fn(),
+        error: jest.fn(),
+        debug: jest.fn(),
+        fatal: jest.fn()
+      }
+    },
+    logTypes: {
```
Comment (jusx) [inline __fixtures__/setup/jestSetUp.js:14]: Do we need to repeat these constants here? Is there a way to reference it from `Logger`?

File: __fixtures__/unit/helper.js [hunk 2]
```diff
@@ -41,9 +41,9 @@ module.exports = {
       log: {
         child: (s) => {
           return {
-            debug: (...s) => console.log(`TEST[debug] > ${JSON.stringify(s)}`),
-            info: (...s) => console.log(`TEST[info] > ${JSON.stringify(s)}`),
-            warn: (...s) => console.warn(`TEST[warn] > ${JSON.stringify(s)}`)
```
Comment (jusx) [inline __fixtures__/unit/helper.js:46]: This was intentionally console.log for debugging purposes while working on the unit tests. Any reason for removing it?

File: __fixtures__/setup/jestSetUp.js (hunk 1 above)
Comment (shine2lay) [reply __fixtures__/setup/jestSetUp.js:14]: I tried many different ways, it seems jest doesn't allow referencing out-of-scope variables. If you have heard of a method. would love to hear it.

File: __fixtures__/unit/helper.js (hunk 2 above)
Comment (shine2lay) [reply __fixtures__/unit/helper.js:46]: there are debugging log every time we ran tests, that's why i replaced it. I just realized we are using logger module, this function is moot. Also, i forgot to use the `Logger` in one file.

File: __fixtures__/setup/jestSetUp.js (hunk 1 above)
Comment (shine2lay) [reply __fixtures__/setup/jestSetUp.js:14]: One solution i can think of is to separate the `logger` from `logTypes`

Comment (shine2lay) [reply __fixtures__/setup/jestSetUp.js]: I implemented it the solution i mentioned

File: __fixtures__/unit/helper.js (hunk 2 above)
Comment (shine2lay) [reply __fixtures__/unit/helper.js:46]: changed the `checkReRun` file to use `Logger` module

**Pull request merged**